app/controllers/phrasing_phrases_controller.rb

```diff
@@ -33,7 +38,8 @@ def download
     app_env = Rails.env
     time = Time.now.strftime('%Y_%m_%d_%H_%M_%S')
     filename = "phrasing_phrases_#{app_name}_#{app_env}_#{time}.yml"
-    send_data Phrasing::Serializer.export_yaml, filename: filename
+    @phrasing_phrases ||= PhrasingPhrase.where("value is not null")
+    send_data Phrasing::Serializer.export_yaml(@phrasing_phrases), filename: filename
   end
```

Comment:
We can get our phrases here with `PhrasingPhrase.fuzzy_search(params[:search], params[:locale])` instead of checking if the instance variable is already set. Then we don't need to hop from one action to another.